tools/internal_ci/helper_scripts/prepare_build_linux_rc

```diff
@@ -31,4 +31,11 @@ PYTHONWARNINGS=ignore XDG_CACHE_HOME=/tmp/xdg-cache-home sudo -E pip install cov
 # Download Docker images from DockerHub
 export DOCKERHUB_ORGANIZATION=grpctesting
 
+# If this is a PR using RUN_TESTS_FLAGS var, then add flags to filter tests
+ if [ -n "$KOKORO_GITHUB_PULL_REQUEST_NUMBER" ] && [ -n "$RUN_TESTS_FLAGS" ]; then
+  sudo apt-get install jq
```

Comment:
It installed without the -y arg, so it might be Kokoro's Ubuntu setting allowing this. I added the -y arg regardless in case this behavior changes in the future.